core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/handler/HasTraitHandler.java

```diff
@@ -34,6 +34,7 @@ public class HasTraitHandler implements OutputOperationHandler<HasTrait, Boolean
     public Boolean doOperation(final HasTrait operation, final Context context, final Store store) throws OperationException {
         Map<String, String> filteredOptions = operation.getOptions().entrySet()
                 .stream()
+                //TODO FS This Handler has FederatedStore logic.
```

Comment:
Do you mean that ideally there shouldn't be fed store logic in here?

Reply:
yes